client/my-sites/importer/newsletter/subscribers.tsx

```diff
@@ -23,19 +21,14 @@ export default function Subscribers( { nextStepUrl, selectedSite, fromSite }: Pr
 				To generate a CSV file of all your Substack subscribers, go to the Subscribers tab and click
 				'Export.' Once the CSV file is downloaded, upload it in the next step.
 			</p>
-			<Button href={ `https://${ fromSite }/publish/subscribers` }>Export subscribers</Button>
+			<Button variant="secondary" href={ `https://${ fromSite }/publish/subscribers` }>
+				Export subscribers
```

Comment:
needs translation

Reply:
We are not translating string just yet. Will do so later.